src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -171,7 +171,7 @@ def add_to_nwbfile(
         write_as: Literal["stimulus", "acquisition"] = "stimulus",
         iterator_options: Optional[dict] = None,
         overwrite: bool = False,
-        verbose: bool = True,
+        verbose: bool = False,
```

Comment:
This verbose is actually unused (and redundant with the one in the init of the same interface), so it should be removed.

Reply:
You are right. Not sure where it is best. I think that I prefer the __init__ one.

Reply:
init is definitely best.  It's not even used in the add_to_nwbfile method at all. Plus all other interfaces define verbose in the init (I checked).

Reply:
Makes sense, removed. 

I also removed the overwrite here. I think this was copied verbatim from some other place or we forgot to update it . 

Thanks for noticing.